runtime/src/main/java/org/corfudb/runtime/view/workflows/WorkflowRequest.java

```diff
@@ -59,20 +59,19 @@
      * orchestrator
      */
     protected ManagementClient getOrchestrator(@NonNull Layout layout) {
-        List<String> activeLayoutServers = layout.getLayoutServers().stream()
-                .filter(s -> !layout.getUnresponsiveServers().contains(s)
-                        && !s.equals(nodeForWorkflow))
+        List<String> availableLayoutServers = layout.getLayoutServers().stream()
+                .filter(s -> !s.equals(nodeForWorkflow))
                 .collect(Collectors.toList());
 
-        if (activeLayoutServers.isEmpty()) {
+        if (availableLayoutServers.isEmpty()) {
             throw new WorkflowException("getOrchestrator: no available orchestrators " + layout);
         }
 
         // Select an available orchestrator
         ManagementClient managementClient = runtime.getLayoutView().getRuntimeLayout(layout)
```

Comment:
what if the selected orchestrator node is actually unresponsive? couldn't we fall into an infinite loop and never complete the workflows?